portal/src/pages/kom-i-gang/lekegrind.tsx

```diff
@@ -0,0 +1,117 @@
+import * as expandButtonReact from "@fremtind/jkl-expand-button-react";
+import * as feedbackReact from "@fremtind/jkl-feedback-react";
+import * as fieldGroupReact from "@fremtind/jkl-field-group-react";
+import * as footerReact from "@fremtind/jkl-footer-react";
+import * as hamburgerReact from "@fremtind/jkl-hamburger-react";
+import * as iconButtonReact from "@fremtind/jkl-icon-button-react";
+import * as iconsReact from "@fremtind/jkl-icons-react";
+import * as imageReact from "@fremtind/jkl-image-react";
+import * as listReact from "@fremtind/jkl-list-react";
+import * as loaderReact from "@fremtind/jkl-loader-react";
+import * as logoReact from "@fremtind/jkl-logo-react";
+import * as messageBoxReact from "@fremtind/jkl-message-box-react";
+import * as radioButtonReact from "@fremtind/jkl-radio-button-react";
+import * as selectReact from "@fremtind/jkl-select-react";
+import * as summaryTableReact from "@fremtind/jkl-summary-table-react";
+import * as tableReact from "@fremtind/jkl-table-react";
+import * as tabsReact from "@fremtind/jkl-tabs-react";
+import * as tagReact from "@fremtind/jkl-tag-react";
+import * as textInputReact from "@fremtind/jkl-text-input-react";
+import * as toggleSwitchReact from "@fremtind/jkl-toggle-switch-react";
+import { motion } from "framer-motion";
+import React, { FC, useState } from "react";
+import { LivePreview, LiveProvider, LiveError } from "react-live";
+import { LiveEditor } from "../../components/interactive-code/LiveEditor";
+import { Seo } from "../../components/seo";
+import { PageTitle } from "../../components/Typography";
+
+const pageTitle = "Lekegrind";
+
+const scope = {
+    ...accordionReact,
+    ...alertMessageReact,
+    ...breadcrumbReact,
+    ...cardReact,
+    ...checkboxReact,
+    ...contentToggleReact,
+    ...coockieConsentReact,
+    ...datepickerReact,
+    ...descriptionListReact,
+    ...expandButtonReact,
+    ...feedbackReact,
+    ...fieldGroupReact,
+    ...footerReact,
+    ...hamburgerReact,
+    ...iconButtonReact,
+    ...iconsReact,
+    ...imageReact,
+    ...listReact,
+    ...loaderReact,
+    ...logoReact,
+    ...messageBoxReact,
+    ...radioButtonReact,
+    ...selectReact,
+    ...summaryTableReact,
+    ...tableReact,
+    ...tabsReact,
+    ...tagReact,
+    ...textInputReact,
+    ...toggleSwitchReact,
+    ...buttonReact,
+};
+
+const defaultCode = `
+    <h2 className="jkl-heading-2 jkl-spacing-s--bottom">Velkommen hit</h2>
+    <p>Her kan du teste Jøkuls komponenter direkte i nettleseren.</p>
+    <p>Bare finn komponenten du vil teste blant Jøkuls komponenter, og lim inn koden i tekstfeltet.</p>
+`;
+
+export const Head: FC = () => <Seo title={pageTitle} />;
+
+const Lekegrind: FC = () => {
+    const [code, setCode] = useState<string>(defaultCode);
+
+    const handleChange = (newCode: string) => {
+        setCode(newCode);
+    };
+
+    const transformCode = (code: string) => {
+        return `<div data-hallo>${code}</div>`;
```

Comment:
Tenker vi fjerner `data-hallo` nå som vi har sett at den funker som forventet

```suggestion
        return `<div>${code}</div>`;
```